test.html

```diff
@@ -166,7 +170,6 @@
             _.delay(callback, 1000, "http://fake.image.url");
         },
         apiOptions: {
-            enableOldAnswerTypes: true,
```

Comment:
Cool! I think we've removed all of the answer areas from questions live on Khan Academy, so this is a good change.

For small changes like this I support making a separate branch/commit and merging that, since it's easy to review on its own, doesn't get held up by a review of the larger change, and makes `git bisect` work better. (Though sometimes it's a lot of pain to extract fixes like these from the rest of the work, which is understandable. It's also not a bad change to make as part of a larger change; it could just probably be served better by separating it.)